packages/website/ts/components/sections/landing/hero.tsx

```diff
@@ -13,26 +15,68 @@ import { WebsitePaths } from 'ts/types';
 //     shouldOpenInNewTab: false,
 // };
 
-export const SectionLandingHero = () => (
-    <Hero
-        title="Powering Decentralized Exchange"
-        isLargeTitle={true}
-        isFullWidth={true}
-        description="0x is an open protocol that enables the peer-to-peer exchange of assets on the Ethereum blockchain."
-        figure={<LandingAnimation image={<HeroAnimation />} />}
-        actions={<HeroActions />}
-        // announcement={announcement}
-    />
-);
+export interface SectionlandingHeroProps {}
+export interface SectionLandingHeroState {
+    isVideoOpen: boolean;
+}
+
+export class SectionLandingHero extends React.Component<SectionlandingHeroProps, SectionLandingHeroState> {
+    public state: SectionLandingHeroState = {
+        isVideoOpen: false,
+    };
+
+    public render(): React.ReactNode {
+        const { isVideoOpen } = this.state;
+        return (
+            <>
+                <Hero
+                    title="Powering Decentralized Exchange"
+                    isLargeTitle={true}
+                    isFullWidth={true}
+                    description="0x is an open protocol that enables the peer-to-peer exchange of assets on the Ethereum blockchain."
+                    figure={<LandingAnimation image={<HeroAnimation />} />}
+                    actions={<HeroActions onPlayVideoClick={this._openModalVideo} />}
+                    // announcement={announcement}
+                />
+                <ModalVideo
+                    channel="youtube"
+                    isOpen={isVideoOpen}
+                    videoId="c04eIt3FQ5I"
```

Comment:
should we put the videoId in constants?

Reply:
Our rule is if it's used twice then it must be a variable, so I think we're good here.